documentation/documentation.css

```diff
@@ -0,0 +1,99 @@
+body{
+    background-color: var(--light);
+}
+.header{
+    grid-area: header;
+    background-color: var(--primary-color);
+    position: fixed;
+    top: 0;
+    z-index: 999;
+}
+.footer{
+    grid-area: footer;
+}
+.main{
+    grid-area: main;
+    margin: 6rem 15rem 2rem 19rem ;
+}
+.nav{
+    grid-area: nav;
+    width: 17rem;
+    padding-left: 2rem;
+    background-color: var(--secondary-nav-color);
+    font-size: 0.9rem;
+    font-family: 'Inter', sans-serif;
+    position: fixed;
+    left: 0;
+    top: 4rem;
+    height: 100%;
+    overflow-y: scroll;
+    z-index: 999;
```

Comment:
give z-index like 10 or something.

Reply:
that is for maximum.. Nothing can come above it